test/unittest/CMakeLists.txt

```diff
@@ -103,6 +103,16 @@ if(is_dpcpp)
   )
 endif()
 
+if( ("${TUNING_TARGET}" STREQUAL "INTEL_GPU") OR
+    ("${DPCPP_SYCL_ARCH}" MATCHES "_pvc|_agm") )
+    message(WARNING "disabling trsv, tbsv and tpsv tests for Intel ARC and Intel GPU Max GPUs")
+    set(TESTS_TO_SKIP
+      ${PORTBLAS_UNITTEST}/blas2/blas2_trsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tbsv_test.cpp
+      ${PORTBLAS_UNITTEST}/blas2/blas2_tpsv_test.cpp
+    )
+endif()
+
```

Comment:
Thank you for the review @pgorlani 

I thought to use both mechanism to avoid issues with tests. I disabled entirely the test for `INTEL_GPU` and specific target to entirely avoid output about them, since we know that right now they are not supported for these specific devices. I catch the exception inside the tests in case portBLAS is built with `DEFAULT` `TUNING_TARGET` to avoid test failure on all possible device. Do you find it redundant?

Reply:
So far, these operators are supported for the integrated Intel GPUs, so they should be tested in case of `INTEL_GPU` target. In case of Arc or Max GPUs, the runtime check skips them as no matter is the tuning target.

Reply:
yes, we can use the run-time checks only. I will remove the cmake checks entirely, even though I would have kept the  check on the arch.

Reply:
Addressed in [75960e5](https://github.com/codeplaysoftware/portBLAS/pull/524/commits/75960e5bd3b523c9850d21feb017f6b3523b5595)